WebDriverAgentTests/IntegrationApp/Classes/FBInvisibleParentScrollingController.m

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBInvisibleParentScrollingController.h"
+#import "FBTableDataSource.h"
+
+static const CGFloat FBSubviewHeight = 40.0;
+
+@interface FBInvisibleParentScrollingController ()
+@property (weak, nonatomic) IBOutlet UIScrollView *scrollView;
+@property (strong, nonatomic) IBOutlet FBTableDataSource *dataSource;
+
+@end
+@implementation FBInvisibleParentScrollingController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self setupLabelViews];
+    self.scrollView.contentSize = CGSizeMake(CGRectGetWidth(self.view.frame), self.dataSource.count * FBSubviewHeight + 800);
+}
+
+- (void)setupLabelViews
+{
+    NSUInteger count = 15;
+    for (NSUInteger i = 0 ; i < count ; i++) {
```

Comment:
I followed what was in `FBScrollViewController`. I was not sure how to do that in `stoyboard`. I updated this part of code same as `FBScrollViewController` as tests were failing on `iPad` as `WDA` label was visible